website/addons/evernote/settings/local-dist.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think adding 

```
EVERNOTE_CLIENT_ID = None
EVERNOTE_CLIENT_SECRET = None
EVERNOTE_SANDBOX = True
```
to  `website/addons/evernote/settings/defaults.py` satisfies this issue.